information_sampling_task/experiment.js

```diff
@@ -153,9 +153,10 @@ var resetRound = function() {
 	colors = jsPsych.randomization.shuffle(['green', 'red', 'blue', 'teal', 'yellow', 'orange',
 		'purple', 'brown'
 	]).slice(0,2)
-	numbersArray = jsPsych.randomization.repeat(numbers, 1)
-	color1_index = numbersArray.slice(0,13)
-	color2_index = numbersArray.slice(13)
+	var numbersArray = jsPsych.randomization.repeat(numbers, 1)
+	var num_majority = Math.floor(Math.random()*5) + 13
```

Comment:
Do you want this to sample from the entire array of numbers? The max it will return is 18, and the numbers array has length 25.